src/components/date/date.component.tsx

```diff
@@ -125,7 +126,7 @@ export const DateInput = React.forwardRef<HTMLInputElement, DateInputProps>(
       onClick,
       onFocus,
       onKeyDown,
-      pickerProps = {},
+      pickerProps = { mode: "single" },
```

Comment:
comment: if setting this to anything but `single` will break the component layout etc, I think rather than setting this as a default we could apply it at the point we pass the props to the DayPicker in a way that prevents it being overridden etc (we should `Omit` it from the pickerProps type as well imo)

in date-picker.component:
```
<DayPicker
   ...
  {...pickerProps}
  mode="single"